### ISSUE TITLE
Image background with vertical slides

### ISSUE THREAD

**Issue by kubamarchwicki** [asciidoctor/asciidoctor-reveal.js#96]
Hi, 
The following Asciidoc

```
= Test slide deck
:imagesdir: images
:backend: revealjs

[%notitle]
== Opening

image::cover.jpg[background, size=cover]

== I'm just title, no background

=== First vertical with background

image::70s.jpg[canvas, size=cover]
```

outputs html

```
<section class="stack future" aria-hidden="true" style="top: 0px; display: block;" hidden="">
  <section style="top: 227px; display: block;" id="i_m_just_title_no_background" data-background-image="images/70s.jpg" data-background-size="cover"> <!--1-->
    <h2>I’m just title, no background</h2>
  </section>
  <section style="top: 330px; display: none;" aria-hidden="true" class="future" id="first_vertical_with_background" data-background-image="images/70s.jpg" data-background-size="cover">
    <h2>First vertical with background</h2>
  </section>
</section>
```

The first subsection has a `data-background-image` set - which I think is obsolete. 
in general the outer section section element has the first image as background - which I consider an error

To work this around you can place whatever empty image as outer section background, but that's rather hack than solution.

**Comment by mojavelinux** [comment]
At first glance, it looks like the new logic to promote a block image to a background isn't considering nested sections correctly.

**Comment by kubamarchwicki** [comment]
That's my idea as well, but I couldn't hack sections template to have it
working :(
Thus only an error report

-- Kuba

**Comment by obilodeau** [comment]
Provided a simple test case :ballot_box_with_check: 

Testing this, be right back!

**Comment by obilodeau** [comment]
Pushed a fix for it in #97.

**Comment by obilodeau** [comment]
Merged #97 which fixes this issue.
